pkg/sfu/buffer/buffer.go

```diff
@@ -349,7 +360,7 @@ func (b *Buffer) calc(pkt []byte, arrivalTime int64) {
 		return
 	}
 
-	b.updateStreamState(&p, len(pkt), arrivalTime, isRTX)
+	flowState := b.updateStreamState(&p, arrivalTime)
```

Comment:
This is actually something I wanted to do at some point. Maybe, this will enable it.

The idea is the publisher figuring out things like if there is loss, how big the loss is, etc and adding that metadata to the forwarded packet. Currently, each down track is checking these things in the munger(s). If publisher can do it once and pass that metadata along, it will be more efficient. We should be able to use this. The `flowState` has information about if there is loss, starting/ending sequence number of loss range.